_locales/fr/messages.json

```diff
@@ -4,12 +4,12 @@
         "description": "Name of the extension."
     },
     "fullName": {
-        "message": "SponsorBlock pour YouTube - Enlève les endossements",
+        "message": "SponsorBlock pour YouTube - Enlève les messages commerciaux et publicités",
```

Comment:
What do you think about `messages commerciaux et publicités intégrés`?

Reply:
"messages commerciaux et publicités intégré**e**s" since ads are "female"